tools/pipelines/templates/build-docker-service.yml

```diff
@@ -336,6 +336,18 @@ extends:
                 echo "$(containerTagSuffix)"
                 echo "##vso[task.setvariable variable=version;isOutput=true]$(containerTagSuffix)"
 
+        # The 1ES.BuildContainerImage task apparently expects to see the CredScan suppressions file not in the root of
```

Comment:
```suggestion
        # The 1ES.BuildContainerImage task expects to see the CredScan suppressions file not in the root of
```

Reply:
I hesitate to apply the change because this was just my educated guess based on the setup that made things work, but I didn't find it documented anywhere. I'd rather keep the hesitation so nobody reads this as an authoritative account of why this is necessary.